### ISSUE TITLE
Today's insider build prevents tasks from ending

### ISSUE THREAD

**Issue by jcasale** [Microsoft/vscode#31387]
- VSCode Version: 1.15.0-insider Date 2017-07-25T07:30:52.414Z
- OS Version: Windows 10.0.14393

Steps to Reproduce:

1. Setup a task such as:
```
{
    "taskName": "MyTask",
    "command": "powershell.exe",
    "args": [
        "-Command",
        "& {...}"
    ],
    "presentation": {
        "reveal": "always",
        "panel": "shared"
    },
    "problemMatcher": []
}
```
2. Run the task, task fails to end.

Reproduces without extensions: Yes

If you cancel the task and try to run any powershell task, the following message is displayed "Terminal with ID 2 does not exist (has it already been disposed?)". Even shell commands fail to end, the task simply hangs.

Where are insider builds accumulated? I have no way to rollback a day when this worked?

**Comment by eamodio** [comment]
This is a pretty big blocker imo -- it stops build tasks from completing and for the first time in a **really** long time, I've had to switch back to using the stable version.

**Comment by Lixire** [comment]
Hey @jcasale , I'm not too sure what you mean by fails to end. Also, what did you put in the curly braces? I tried echo and the task seemed to have no problems.

**Comment by jcasale** [comment]
vscode refuses to start another task, it says:
```The task '{0}' is already active. To terminate it use `Terminate Task...` from the Tasks menu.```
Yes it fails to format the string. The code in the braces is irrelevant, no task of any sort will start after the first. In this case it was a powershell command removing and recreating a build tmp directory. Terminate task reveals the same task as active, however the interpreter does not appear in a task list.

**Comment by eamodio** [comment]
This is resolved now for me on the latest insiders `1.15.0-insider` 0e7546c978fa1b62dbf36f07a77d1b9a4241d6f7

**Comment by jcasale** [comment]
Same, thanks for the follow up.
